test/src/test/java/jenkins/BreadCrumbTest.java

```diff
@@ -0,0 +1,15 @@
+package jenkins;
+
+import org.junit.Test;
+import static org.junit.Assert.assertNotNull;
+
+
+public class BreadCrumbTest {
+
+    @Test
+    public void testNonNullBreadCrumbItemList() {
+        //The breadcrumb Items list  would at least have the home root breadcrumb
+        assertNotNull(new BreadCrumb().generateBreadCrumbs());
```

Comment:
I am not too sure how to proceed with a test for this, do I need to setup some sort of a mock jenkins server to run and then check the root breadcrumbs

Reply:
This throws me a ton of errors :disappointed: